scripts/performance/benchmark.sh

```diff
@@ -20,49 +20,48 @@ S3_RESULTS_DIR="performance-test/results"
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 GIT_HASH="$(git rev-parse --short HEAD)"
 
-
 # Save the results filename to a temporary file
 RESULTS_FILENAME_FILE=$(mktemp)
 trap 'rm -f $RESULTS_FILENAME_FILE' EXIT
 
 function read_benchmark_logs_for_results() {
-    if [[ $line =~ Results\ saved\ to:\ ([^\ ]+) ]]; then
-        results_filename="${BASH_REMATCH[1]}"
-        echo "CSV file value found: $results_filename"
-        echo "$results_filename" > "$RESULTS_FILENAME_FILE"  # Store the value in the temporary file
-    fi
+	if [[ $line =~ Results\ saved\ to:\ ([^\ ]+) ]]; then
+		results_filename="${BASH_REMATCH[1]}"
+		echo "CSV file value found: $results_filename"
+		echo "$results_filename" >"$RESULTS_FILENAME_FILE" # Store the value in the temporary file
+	fi
 }
 
 if [[ "$DOCKER_TEST" == "true" ]]; then
-    DOCKER_IMAGE=unstructured:perf-test make docker-build 
-    docker rm -f unstructured-perf-test > /dev/null 2>&1
-    docker run \
-    --name unstructured-perf-test \
-    --rm \
-    -e NUM_ITERATIONS="$NUM_ITERATIONS" \
-    -e INSTANCE_TYPE="$INSTANCE_TYPE" \
-    -e GIT_HASH="$GIT_HASH" \
-    -e SLOW_FILES="${SLOW_FILES[*]}" \
-    -e HI_RES_STRATEGY_FILES="${HI_RES_STRATEGY_FILES[*]}" \
-    -v "${SCRIPT_DIR}":/home/notebook-user/scripts/performance \
-    unstructured:perf-test \
-    bash /home/notebook-user/scripts/performance/benchmark-local.sh 2>&1 | tee >(while IFS= read -r line; do
-        read_benchmark_logs_for_results
-    done)
+	DOCKER_IMAGE=unstructured:perf-test make docker-build
+	docker rm -f unstructured-perf-test >/dev/null 2>&1
+	docker run \
+		--name unstructured-perf-test \
+		--rm \
+		-e NUM_ITERATIONS="$NUM_ITERATIONS" \
+		-e INSTANCE_TYPE="$INSTANCE_TYPE" \
+		-e GIT_HASH="$GIT_HASH" \
+		-e SLOW_FILES="${SLOW_FILES[*]}" \
+		-e HI_RES_STRATEGY_FILES="${HI_RES_STRATEGY_FILES[*]}" \
+		-v "${SCRIPT_DIR}":/home/notebook-user/scripts/performance \
+		unstructured:perf-test \
+		bash /home/notebook-user/scripts/performance/benchmark-local.sh 2>&1 | tee >(while IFS= read -r line; do
+			read_benchmark_logs_for_results
+		done)
 else
-    NUM_ITERATIONS="$NUM_ITERATIONS" INSTANCE_TYPE="$INSTANCE_TYPE" GIT_HASH="$GIT_HASH" SLOW_FILES="${SLOW_FILES[*]}" HI_RES_STRATEGY_FILES="${HI_RES_STRATEGY_FILES[*]}" "$SCRIPT_DIR"/benchmark-local.sh 2>&1 | \
-    tee >(while IFS= read -r line; do
-        read_benchmark_logs_for_results
-    done)
+	NUM_ITERATIONS="$NUM_ITERATIONS" INSTANCE_TYPE="$INSTANCE_TYPE" GIT_HASH="$GIT_HASH" SLOW_FILES="${SLOW_FILES[*]}" HI_RES_STRATEGY_FILES="${HI_RES_STRATEGY_FILES[*]}" "$SCRIPT_DIR"/benchmark-local.sh 2>&1 |
```

Comment:
I don't know shell well enough to tell if the backslash can be removed...

Reply:
verified this works